controllers/assignments.py

```diff
@@ -529,13 +528,13 @@ def _autograde_one_q(course_name, assignment_id, sid, qname, points, deadline=No
     question = db(db.questions.name == qname).select().first()
 
     # dispatch on grading_type; if none specified, can't autograde
-    if question.autograde == 'unittest':
+    if autograde == 'unittest':
```

Comment:
We talked about it this summer.  The path is where the instructor selects Chapter for the first column, a question from the chapter for the second, and then student in the third column.